### PR TITLE
Additions to heron/common for python topology

### PR DESCRIPTION
**Pull request by taishi8117**
- Added custom serializer support

### CODE REVIEW

File: heron/common/src/python/utils/misc/serializer.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 except:
   import pickle
 
-class HeronSerializer(object):
+class IHeronSerializer(object):
```
Comment (kramasamy) [inline heron/common/src/python/utils/misc/serializer.py]: Why change to IHeronSerializer?

**Comment by kramasamy** [comment]
👍

File: heron/common/src/python/utils/misc/serializer.py (hunk 1 above)
Comment (objmagic) [inline heron/common/src/python/utils/misc/serializer.py]: also wondering... we don't have to even follow the naming convention from Factory method of Java

Comment (taishi8117) [inline heron/common/src/python/utils/misc/serializer.py]: I thought it might be better to indicate in the name that this is an interface.

**Comment by objmagic** [comment]
👍

**Pull request merged**